index.html

```diff
@@ -424,6 +444,84 @@ <h2 class="experience-text">Skills</h2>
           </li>
         </ul>
       </section>
+
+      <!-- Contact 4/4 - Layout -->
+
+      <section class="l-contact">
+        <!-- Background Images -->
+
+        <div class="e-12"></div>
+        <div class="e-3"></div>
+        <div class="e-5"></div>
+        <div class="r-7"></div>
+        <div class="r-7 r-7-b"></div>
+
```

Comment:
- [ ] Please make sure to use the same background image as the given design in order to follow the exact layout.
Kindly notice that styling the background as an image is a lot easier than hard coding it the way you did with HTML and CSS. It helps to keep the code cleaner and optimized.

_Please look at the images down below to see the differences, then make the corresponding changes._
> **Quick Note:** Please make sure that you follow the template in typography, colors, font size, font weight, images, and spacings (margin and padding).

### **Your design**
![Screenshot1](https://user-images.githubusercontent.com/47625407/121219180-b8b3e380-c883-11eb-834e-b99ec68eb743.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)**
![Screenshot](https://user-images.githubusercontent.com/47625407/121219224-c4070f00-c883-11eb-8088-805381a0ba5b.png)

Reply:
I had a weird problem with the SCSS file. For some reason, the nested properties were ignored. I deleted '_contact.scss' and pasted a copy that I got stored and everything worked right.

Reply:
I couldn't understand what is it that you're doubting?
You clearly answered yourself 😅👀